Homework/Homework5/Actions.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace Homework5
+{
+    public class Actions
+    {
+
+        public static Result FirstMethod()
+        {
+            Logger.Instance.Log(LogLevel.Info, "Start method: FirstMethod");
+            return new Result();
```

Comment:
For what reason we need emptu Result then?

Reply:
This Class has no logic :( Because I don't clearly understand required